src/template.rs

```diff
@@ -18,22 +26,50 @@ pub fn generate_template(name: &str, git: Option<&str>) -> anyhow::Result<PathBu
 
 /// Returns the [`TemplatePath`] for a given Git repository.
 ///
-/// If `git` is [`None`], it will default to `bevy_quickstart`.
-fn template_path(git: Option<&str>) -> TemplatePath {
-    // Use a minimal template by default.
-    const DEFAULT_REPOSITORY: &str = "https://github.com/TheBevyFlock/bevy_new.git";
-    const DEFAULT_BRANCH: &str = "main";
-
-    if let Some(template) = git {
-        TemplatePath {
-            git: Some(template.to_string()),
-            ..Default::default()
-        }
-    } else {
-        TemplatePath {
-            git: Some(DEFAULT_REPOSITORY.to_string()),
-            branch: Some(DEFAULT_BRANCH.to_string()),
-            ..Default::default()
+/// If `template` is [`None`], it will default to `bevy_new_default`.
+fn template_path(template: &str, branch: &str) -> anyhow::Result<TemplatePath> {
+    const TEMPLATE_ORG: &str = "TheBevyFlock";
+    const TEMPLATE_PREFIX: &str = "bevy_new_";
+
+    let templates = fetch_template_repositories(TEMPLATE_ORG, TEMPLATE_PREFIX)?;
+    let maybe_url = templates.iter().find_map(|r| {
+        // Does the provided argument match any of our existing templates?
+        let suffix = &r.name[TEMPLATE_PREFIX.len()..];
+        if suffix == template {
+            return Some(r.html_url.clone());
         }
-    }
+        None
```

Comment:
Can be simplified using `(suffix == template).then`

Reply:
Ooo, nice.